ddl/reorg.go

```diff
@@ -191,6 +195,14 @@ type reorgInfo struct {
 	PhysicalTableID int64
 }
 
+func (r *reorgInfo) String() string {
+	return "StartHandle:" + strconv.FormatInt(r.StartHandle, 10) + "," +
```

Comment:
Why not use `strconv.Itoa`

Reply:
`StartHandle` is `int64`, `strconv.Itoa` need type `int`. 
Actually `strconv.Itoa` also use `strconv.FormatInt`.